shared/clinical-code-sets/patient/first-language/1/README.md

```diff
@@ -0,0 +1,15 @@
+# First language
```

Comment:
I see there are also codes for "language read". I don't think that applies here because this is about first language rather than what language people can also read - but might be interesting to see if they are widely used on the grounds that if you put "Reads Armenian" into a person's clinical record it probably means that it is their first language.